test/integration/openid4ci_test.go

```diff
@@ -0,0 +1,43 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/openid4ci"
+	"github.com/trustbloc/wallet-sdk/test/integration/pkg/setup/oidc4ci"
+	"github.com/trustbloc/wallet-sdk/test/integration/pkg/testenv"
+)
+
+// Add run this lines to make test working locally
```

Comment:
`Add run`? Do you must just `run`? Or do you mean that these should be added somewhere specific? Also, should be `work` instead of `working`